code/Flexbox/index.html

```diff
@@ -0,0 +1,94 @@
+ <!DOCTYPE html>
+ <html lang="en">
+ <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Flexbox Challenge</title>
+    <link href="style.css" rel="stylesheet" type="text/css" />
+ </head>
+ <body>
+    <h1> CSS Flexbox</h1>
+
+   <h3> Challenge 1</h3>
+    <p> Horizontally list these bubbles using Flexbox </p>
+    <div class="blue bubble"></div>
```

Comment:
You actually got the CSS right, but what went wrong is there should be another opening div here with the class challenge-1. 

I'm marking it as complete though, this is close enough! Nice work.